qemu/tests/fio_windows.py

```diff
@@ -34,6 +36,12 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     session = vm.wait_for_login(timeout=timeout)
+    session_1 = vm.wait_for_login(timeout=timeout)
+
+    if params.get("format_disk", "no") == "yes":
```

Comment:
I have check the test steps, not sure why do you need to format disk here

Reply:
hi hachen, 

it because that we need to run fio read and write test on sys&data disk.  if the disk is not formatted, it only support fio read operation, don't support fio write operation on it. if the disk is formatted, it can support both fio read and write, so need format here.

Reply:
if format disk is required here, I would's suggest use  `params["format_disk"]`, which indicates that this param is not optional here.

Reply:
Maybe you can remove `format_disk` in the cfg , and always format disk it is required for all cases